server_installation/topics/operating-mode/crossdc.adoc

```diff
@@ -132,26 +137,25 @@ For more detail about how caches can be configured see <<tuningcache>>.
 [[communication]]
 ==== Communication details
 
-{project_name} uses multiple, separate clusters of Infinispan caches. Every {project_name} node is in the cluster with the other {project_name} nodes in same data center, but not with the {project_name} nodes in different data centers. A {project_name} node does not communicate directly with the {project_name} nodes from different data centers. {project_name} nodes use external JDG (actually {jdgserver_name} servers) for communication across data centers. This is done using the link:https://infinispan.org/docs/8.2.x/user_guide/user_guide.html#using_hot_rod_server[Infinispan HotRod protocol].
+{project_name} uses multiple, separate clusters of Infinispan caches. Every {project_name} node is in the cluster with the other {project_name} nodes in same data center, but not with the {project_name} nodes in different data centers. A {project_name} node does not communicate directly with the {project_name} nodes from different data centers. {project_name} nodes use external {jdgserver_name} (actually {jdgserver_name} servers) for communication across data centers. This is done using the link:https://infinispan.org/docs/8.2.x/user_guide/user_guide.html#using_hot_rod_server[Infinispan HotRod protocol].
 
-The Infinispan caches on the {project_name} side must be configured with the link:https://infinispan.org/docs/8.2.x/user_guide/user_guide.html#remote_store[remoteStore] to ensure that data are saved to the remote cache. There is separate Infinispan cluster between JDG servers, so the data saved on JDG1 on `site1` are replicated to JDG2 on `site2` .
+The Infinispan caches on the {project_name} side must be configured with the link:https://infinispan.org/docs/8.2.x/user_guide/user_guide.html#remote_store[remoteStore] to ensure that data are saved to the remote cache. There is separate Infinispan cluster between {jdgserver_name} servers, so the data saved on JDG1 on `site1` are replicated to JDG2 on `site2` .
 
-Finally, the receiving JDG server notifies the {project_name} servers in its cluster through the Client Listeners, which are a feature of the HotRod protocol. {project_name} nodes on `site2` then update their Infinispan caches and the particular user session is also visible on {project_name} nodes on `site2`.
+Finally, the receiving {jdgserver_name} server notifies the {project_name} servers in its cluster through the Client Listeners, which are a feature of the HotRod protocol. {project_name} nodes on `site2` then update their Infinispan caches and the particular user session is also visible on {project_name} nodes on `site2`.
```

Comment:
Client listeners aren't a feature of the Hot Rod protocol. 

When you add data to a cache with a remote store configured in it, the data is transmitted to the remote DG cluster over Hot Rod. In this way the data in the cache is externalized to the DG cluster.

Clusters communicate across sites using JGroups RELAY2. When you add data to 'site1' one or more nodes in that cluster replicate the data to one or more nodes at 'site2'.

(Also please note Hot Rod is two separate words)

Reply:
Client listeners are a feature of Hot Rod. When site2 receives an update from site1, the server triggers an event. The client listener is notified and keycloack server can update their embedded cache with the new value.

Reply:
Thanks for keeping me honest there, Pedro. Actually just re-reading this and it makes a bit more sense so maybe disregard my comment there, Andy. 

Although s/HotRod/Hot Rod